python/tvm/relay/tensorrt.py

```diff
@@ -412,6 +476,20 @@ def pad_whitelist_fn(attrs, args):  # pylint: disable=unused-variable
             return False
         return True
 
+    _register_external_dynamic_check_func("nn.dense", dense_whitelist_fn)
```

Comment:
Can we have all of the `_register_external_dynamic_check_func` calls in the same place at the end?